Dockerfile

```diff
@@ -44,29 +45,28 @@ RUN apt install -y libxkbcommon0 \
     xvfb
 
 COPY ./deploy/requirements/* /opt/
-RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10
-RUN pip3 install -r /opt/click.txt -r /opt/prod.txt
-COPY ./deploy/oz/run-full-test-suite.sh /opt/neon-tests/
+
+RUN curl -sS https://bootstrap.pypa.io/get-pip.py | python3.10 && \
```

Comment:
better install by oneliner from official doc: curl -LsSf https://astral.sh/uv/install.sh | sh
it should be faster (i think)

Reply:
Changed

Reply:
This method isn't working in docker, as the script is relying on `/proc/self/exe`, which is not available:
```
head: error reading '/proc/self/exe': Bad file descriptor
head: failed to close '/proc/self/exe': Bad file descriptor
ERROR: unknown platform bitness
```
I tried to change it to the `/bin/sh`, in that case uv it not seen after installation. I can try to dig further, add it manually to the environment and fix all other issues if any. But I think the more reliable way is to use pip for uv installation, in case developers change something, it will be supported.

Reply:
ok